Doc/library/shutil.rst

```diff
@@ -83,6 +83,10 @@ Directory and files operations
       copy the file more efficiently. See
       :ref:`shutil-platform-dependent-efficient-copy-operations` section.
 
+   .. versionchanged:: 3.14
+      Copy-on-write or server-side copy may be used internally on supported
+      filesystems via :func:`os.copy_file_range`.
+
```

Comment:
I think you only need one of these `.. versionchanged::` directives, at the end of the "Platform-dependent efficient copy operations" section.

Reply:
Thanks for the suggestion, I updated docs in e8feaca42f23310cc1f01bcb20b67ab38f05f9ef